.github/workflows/ci.yml

```diff
@@ -54,6 +54,7 @@ jobs:
         run: pnpm run lint
       - run: pnpm run lint:fs
       - run: pnpm run lint:deps
+      - run: pnpm run lint:rac
       - run: pnpm run lint:package
```

Comment:
<img width="1890" height="1000" alt="image" src="https://github.com/user-attachments/assets/dabc09ce-f0cf-4d6b-aac9-88f106087ef3" />
YOU get linted. YOU get linted. YOU get linted!